library/log/src/modules/LogViewCore.rb

```diff
@@ -153,10 +155,7 @@ def Start(widget, d)
               lines_count = 0 if Ops.less_than(lines_count, 0)
 
               if Ops.greater_than(lines_count, 0)
-                command = Ops.add(
-                  command,
-                  Builtins.sformat(" | tail -n +%1", lines_count)
-                )
+                command << Builtins.sformat(" | tail -n +%1", lines_count.to_s.shellescape)
```

Comment:
Do we really need to shell-escape a number? and while we are at it, why not use Ruby variable expansion here and get rid of this `sformat()`?

Reply:
@shundhammer it depends how paranoic you are and if you believe that lines_count is always only integer or if you think that attacker can somehow pass malicious string there.

Reply:
OK.